src/js/profiling/integration.ts

```diff
@@ -172,16 +195,16 @@ export const hermesProfilingIntegration: IntegrationFn = () => {
   };
 
   const _createProfileEventFor = (profiledTransaction: Event): ProfileEvent | null => {
-    const profile_id = profiledTransaction?.contexts?.['profile']?.['profile_id'];
+    const profile_id = profiledTransaction?.contexts?.['trace']?.['data']?.['profile_id'];
```

Comment:
Should we call `profiledTransaction` as `profiledEvent` or `profiledSpan` since transactions no longer exist?

Reply:
Yes, make sense, I agree, but let's do it in another PR, to avoid adding more change to this, it's already a large PR.